src/routes/Devilstable.svelte

```diff
@@ -33,23 +32,17 @@
 	} = $props();
 
 	let mobile = $derived(width < 800);
-	let [fractions, noFractions] = $derived(
-		data.reduce(
-			/**
-			 *
-			 * @param {[{values: number[][], label: string}[],{values: number[][], label: string}[]]} acc
-			 * @param item
-			 */
-			(acc, item) => {
-				// Determine which sub-array to push the item into based on whether it includes the substring.
-				item.label.includes('fr') ? acc[0].push(item) : acc[1].push(item);
-				return acc;
-			},
-			[[], []] // Initial accumulator value is two empty arrays.
-		)
+	const allowStringsWithoutFragments = ['Fassung', ...codices.map((c) => c.sigil), 'fr'];
```

Comment:
I think the naming of the variable prevents a future coder from just changing this value when they want the default to deviate from the allowed strings. I'm skipping this suggestion because I thinks it's slightly more simplistic. But I really like the suggestion anyways because it shows an eye for future caveats an traps! great comment!

Reply:
I see your point, but still I'd prefer explicitness over simplicity in this case. Another benefit of the suggested change would be that you could omit introducing `allowStringsWithoutFragments` alltogether and just write everything into `whitelist` (racist terminology by the way, but I know it's baked into Skeleton). The only reason for having a `allowStringsWithoutFragments` is the default values...

Reply:
that convinced me! :-)